R/cache.R

```diff
@@ -252,12 +252,12 @@ cache_epidata_call <- function(epidata_call, fetch_args = fetch_args_list()) {
     if (as_of_recent || issues_recent) {
       cli::cli_warn(
         c(
-          "using cached results with `as_of` within the past week (or the future!).",
-          "This will likely result in an invalid cache. Consider\n",
-          "1. disabling the cache for this session with `disable_cache` or permanently with environmental ",
-          "variable `EPIDATR_USE_CACHE=FALSE`\n",
-          "2. setting `EPIDATR_CACHE_MAX_AGE_DAYS={Sys.getenv('EPIDATR_CACHE_MAX_AGE_DAYS', unset = 1)}`",
-          " to e.g. `3/24` (3 hours)."
+          "Using cached results with `as_of` within the past week (or the future!).
+          This will likely result in an invalid cache. Consider",
+          "i" = "disabling the cache for this session with `disable_cache` or
```

Comment:
comment: "i" = is "info item", not numbered list, if that was what you were expecting.  Here it might look prettier to use "*" for a bulleted list.  (Not sure how to do an actual enumeration, but here just an unordered list seems more suitable.)